### PR TITLE
Ramico/cust prototype

### PR DESCRIPTION
**Pull request by ramico**
It works but you should implode it

### CODE REVIEW

File: includes/class-wc-siftscience-admin.php [hunk 1]
```diff
@@ -292,6 +303,16 @@ private function get_section_fields( $sub_section ) {
 						)
 					),
 
+					$this->wc_element->create(
+						WC_SiftScience_Element::CUSTOM,
+						'Cid',
+						'disc label',
+						implode( '|', $this->status->get_status_options() ),
+						array(
+							'callback' => 'gb_callback',
```
Comment (nabsul) [inline includes/class-wc-siftscience-admin.php:312]: Instead of using implode/explode, you can just place the array as a new entry here.

Comment (ramico) [reply includes/class-wc-siftscience-admin.php:312]: ?

File: includes/class-wc-siftscience-element.php [hunk 2]
```diff
@@ -184,7 +184,7 @@ public function gb_callback( $data ) {
 					<?php
 					echo 'default=' . esc_attr( $default ) . ' name=' . esc_attr( $name ) . ' value=' . esc_attr( $value )
 					?>
-					/>
+					/>&nbsp;
```
Comment (ramico) [inline includes/class-wc-siftscience-element.php:187]: Untitled

File: includes/class-wc-siftscience-element.php [hunk 3]
```diff
@@ -150,11 +149,10 @@ public function create( $type, $id, $label = '', $desc = '', $element_options =
 		 * @param Array $data The data array of this setting line.
 		 */
 		public function anon_id_callback( $data ) {
-			$title = $data['title'];
```
Comment (ramico) [inline includes/class-wc-siftscience-element.php:153]: sigle variable used only once (unneeded)

**Pull request merged**